OpenMEEG/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake ${CMAKE_SOURCE_DIR}/cmake/Fortra
 
 project(OpenMEEG LANGUAGES C CXX ${PROJECT_VERSION})
 
+# Tell CMake to use rpath with the libs we build
+set(CMAKE_MACOSX_RPATH 1)
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+
 # The next two lines are a kludge to make rpm generation work. If you change this,
 # please check that rpm generation still works.
```

Comment:
you should put thes 3 lines in OpenMEEGOption.cmake, it gets included in both CMakeLists.txt (through Setup.cmake)